app/models/drawing.rb

```diff
@@ -49,6 +52,14 @@ def self.to_csv(hxl: false)
     end
   end
 
+  def self.stages_of_journey
+    @stages_of_journey = ["At home", "In temporary shelter", "Awaiting transit", "On the move", "Arrived at destination"]
```

Comment:
After discussing with Kris last night on the most Ruby/Rails way of doing this, can you change these to reference just the constant STAGES_OF_JOURNEY, and move the array back up to line 2?

Line 2:
```ruby
STAGES_OF_JOURNEY = ["At home", "In temporary shelter", "Awaiting transit", "On the move", "Arrived at destination"].freeze
```

Line 55-57:
```ruby
def self.stages_of_journey
  STAGES_OF_JOURNEY
end
```